browser/components/EditProject.tsx

```diff
@@ -257,6 +275,17 @@ export default class EditProject extends React.Component<Props, State> {
                   />
                 </div>
 
+                <div className="form-group">
+                  <label>Is GitHub Org? </label>
```

Comment:
Presentation nitpick: Checkbox labels go after the box and have a different group structure:

https://getbootstrap.com/docs/4.3/components/forms/#checkboxes-and-radios

Reply:
Still earning this stuff. Will change.

Reply:
Check the class names again -- this is form-check.

Reply:
It is a form-check but I don't like how it alters the alignment to the other options.